docs/configuration-files/add-configuration-file.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Create configuration file
```

Comment:
The file name is prefixed with `add` operation, here title mentions `Create` operation.
Perhaps adopt a single terminology to make it consistent for the reader?

Reply:
```suggestion
title: Add configuration file
```